common/presenters/message-banner/assessment-to-unconfirmed-banner.js

```diff
@@ -59,17 +59,17 @@ module.exports = function assessmentToUnconfirmedBanner({
     `
   }
 
-  if (!assessment.editable) {
-    content += `
-      ${componentService.getComponent('govukWarningText', {
-        text: i18n.t('messages::assessment.completed.uneditable', {
-          context,
-        }),
-        classes: 'govuk-!-padding-top-0 govuk-!-margin-top-5',
-        iconFallbackText: 'Warning',
-      })}
-    `
-  }
+  // if (!assessment.editable) {
+  //   content += `
+  //     ${componentService.getComponent('govukWarningText', {
+  //       text: i18n.t('messages::assessment.completed.uneditable', {
+  //         context,
+  //       }),
+  //       classes: 'govuk-!-padding-top-0 govuk-!-margin-top-5',
+  //       iconFallbackText: 'Warning',
+  //     })}
+  //   `
+  // }
```

Comment:
I'm assuming this will be deleted for the final version?

Reply:
Doh! Good spot!